recipes/pytfa/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "pytfa" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/EPFL-LCSB/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  #url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b4d848bd04cf8384b05e82fecb25a7cc59a65031ac5dee2c732853a12b086014
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - equilibrator-api
+    - equilibrator-cache
+    - networkx
+    - pandas
+    - cobra >0.13
+    - bokeh >=0.12.1
+    - numpy ==1.22.0
+    - optlang
+    - scipy
+    - tqdm
+    - python-libsbml
```

Comment:
Do you need these in `host`? Nominally `python` and `pip` should be enough for pure Python packages:
```suggestion
```

Reply:
Hmm yes I was not sure. This package should be pure python.